kbs/config/kubernetes/base/deployment.yaml

```diff
@@ -12,6 +12,19 @@ spec:
       labels:
         app: kbs
     spec:
+      initContainers:
+      - command:
+        - sh
+        - -c
+        - cp -r /config/$(dirname $(readlink /config/policy.rego))/* /opa/confidential-containers/kbs/
```

Comment:
This doesn't support updates at runtime or provisioning resources, right? Maybe this is fine in the short term.